tests/behat/features/manage-files.feature

```diff
@@ -63,6 +63,7 @@ Feature: Manage files
 
   Scenario: I can filter the files list view using name
     Given I expand the content filters
+    And I wait for 2 seconds
```

Comment:
I see why that's required (animation). But it makes the test less readable, and makes the test even slower to run. Should be solved by having the previous step ("expand the content filters") waiting for animation to finish. Roughly `$this->waitFor('jQuery(':animated').length == 0')`. Do you want to have a crack at it?